brave/browser-laptop
# add off-by-default application firewall feature

fix https://github.com/brave/browser-laptop/issues/12671
mitigate #12616 

test plan:
1. download an image and name it to rabbits.jpg
2. in the rabbits.jpg directory, start a localhost server: 'python -m SimpleHTTPServer 8000'
3. go to https://jsfiddle.net/c6y5qx5m/. you should see either 2 or
   3 copies of rabbits.jpg loaded.
4. go to about:preferences#security and enable 'Application Firewall'
5. go to https://jsfiddle.net/c6y5qx5m/ in a new private or session tab
   (to avoid loading cached files). now none of the rabbits.jpg images
   should load.

## Submitter Checklist:


## Test Plan:


## Reviewer Checklist:


Tests
